src/chains/migrations/0023_create_gas_price_model.py

```diff
@@ -0,0 +1,75 @@
+# Generated by Django 3.2.6 on 2021-09-01 12:38
+
+import django.db.models.deletion
+import gnosis.eth.django.models
+from django.db import migrations, models
+
+
+def copy_gas_prices(apps, schema_editor):
+    GasPrice = apps.get_model("chains", "GasPrice")
+    Chain = apps.get_model("chains", "Chain")
+    db_alias = schema_editor.connection.alias
+
+    GasPrice.objects.using(db_alias).bulk_create(
```

Comment:
Just curious, is this `using` needed?

Reply:
I would say that it is not required reading this: https://docs.djangoproject.com/en/3.2/ref/models/querysets/#using

Since we are using just one DB. I think we can remove it 👍

Reply:
Removed it here 60342d3